### PR TITLE
Add analytics guide

### PR DESCRIPTION
**Pull request by jasonmp85**
For helping with ParadeDB's testing of this stack

Keeping as draft until this stack goes GA.

### CODE REVIEW

**Comment by samay-sharma** [comment]
Can you add a couple other example queries in the guide and also report the timing maybe so people feel that it's "fast"

File: src/content/docs/product/stacks/Analytical/analytics.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+---
+title: Analytics
+sideBarTitle: Analytics
+sideBarPosition: 305
+tags: [postgresql, paradedb, analytics]
+---
+
+Tembo's Analytics stack enables you to efficiently query large amounts of "off-site" data (such as S3 parquet files) easily. The stack is tuned for analytics workloads.
```
Comment (ChuckHend) [inline src/content/docs/product/stacks/Analytical/analytics.md:8]: We should add here something like "powered by ParadeDB"

**Review by ChuckHend — APPROVED** [review]
Add the queries as requested by @samay-sharma and also add a note such as "powered by ParadeDB" to the intro, then this is good to go in my opinion.

File: src/content/docs/product/stacks/Analytical/analytics.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+First, create a local table and populate it with the same data from S3.
+
+```sql
+CREATE TABLE local_trips (LIKE trips INCLUDING ALL);
+INSERT INTO local_trips SELECT * FROM trips;
+```
+
+Then run the same query on the local Postgres table.
+ Notice the execution time being approximately 3x faster it is enabled by DuckDB.
```
Comment (samay-sharma) [inline src/content/docs/product/stacks/Analytical/analytics.md:134]: Isn't it almost 5x faster? ;)

File: src/content/docs/product/stacks/Analytical/analytics.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+          20 |        159989
+          21 |        160888
+          22 |        143261
+          23 |        109287
+(24 rows)
+
+Time: 3317.911 ms (00:03.318)
+```
+
+## Other Formats
```
Comment (samay-sharma) [inline src/content/docs/product/stacks/Analytical/analytics.md:177]: Other Formats and Stores

Combine both of the sections in one.

**Review by samay-sharma — APPROVED** [review]

File: src/content/docs/product/stacks/Analytical/analytics.md (hunk 2 above)
Comment (ChuckHend) [reply src/content/docs/product/stacks/Analytical/analytics.md]: Well, I was trying to be conservative and rounding the remote storage up to 1 second and round the on-postgres down to 3 seconds.

File: src/content/docs/product/stacks/Analytical/analytics.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+```sql
+SELECT 
+  EXTRACT(HOUR FROM tpep_pickup_datetime) AS pickup_hour,
+  COUNT(*) AS total_records
+FROM trips
+GROUP BY pickup_hour
+ORDER BY pickup_hour;
+```
+
+```timing
```
[the remark this answers is not in the document]
Comment (ChuckHend) [reply src/content/docs/product/stacks/Analytical/analytics.md:90]: lol. it is a harmless typo but good catch. ill change it to `plaintext`, which is what it defaults to.

**Pull request merged**